fizzbuzz.rb

```diff
@@ -10,3 +10,12 @@
 #   otherwise
 #       return the original number (as a string)
 
+def fizz_buzz(number)
+  if(number == 3)
+    return "Fizz"
+  elsif (number == 5)
+    return "Buzz"
+  end
+
+  return number.to_s
```

Comment:
**question:** I am not sure what this does. What is "to_s"?

Reply:
Ah yeah, that's a bit of a poor choice by Ruby — it essentially converts an integer to a string, but is in my opinion, named poorly. I would much prefer it were called `.to_string`